.github/workflows/lint.yml

```diff
@@ -0,0 +1,58 @@
+name: Lint
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  run-linters:
+    name: Run linters
+    runs-on: ubuntu-latest
+    env:
+      NPM_VERSION: "8.3.0"
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14'
+          cache: 'npm'
+          cache-dependency-path: 'frontend/package-lock.json'
+
+      - name: Install dependencies
+        run: npm install --global npm@${{ env.NPM_VERSION }}
+
+      - name: "Install dependencies: workspace"
+        working-directory: frontend
+        run: |
+          npm install
+          # Work-around until Hydrogen is updated: we trick hydrogen into thinking its installed in the package path.
+          # See: https://github.com/hydrogen-design-system/hydrogen.css/issues/72
+          mkdir -p admin/node_modules common/node_modules talentsearch/node_modules
+          (cd admin/node_modules && ln -s ../../node_modules/@hydrogen-design-system .)
+          (cd common/node_modules && ln -s ../../node_modules/@hydrogen-design-system .)
+          (cd talentsearch/node_modules && ln -s ../../node_modules/@hydrogen-design-system .)
```

Comment:
I think this might no longer be necessary? @patcon ?

Reply:
ok let me try removing it

Reply:
That's my understanding! The root issue is fixed in https://github.com/hydrogen-design-system/hydrogen.css/issues/72 🚀

Reply:
can be removed in other places in this PR too!